src/TipoffPackage.php

```diff
@@ -96,4 +96,11 @@ public function hasNovaResources(array $novaResources): self
 
         return $this;
     }
+
+    public function hasBindings(array $bindings): self
+    {
+        $this->bindings = array_merge($this->bindings, $bindings);
```

Comment:
Like other types, `bindings` should be declared as a class property, initialized to empty array.